retry.go

```diff
@@ -0,0 +1,80 @@
+package pixela
+
+import (
+	"math"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/pkg/errors"
+)
+
+const maxRetryCount = 20
+
+// RetryCount number of retries when an API call is rejected (max: 20)
+var RetryCount = 0
+
+// ErrAPICallRejected API call rejected.
+// See: https://help.pixe.la/en/blog/release-request-rejecting
+var ErrAPICallRejected = errors.New("api call rejected")
+
+type retryer struct {
+	processFunc func(r *retryer)
+	maxRetry    int
+	statusCode  int
+	body        []byte
+	err         error
+}
+
+func (m *retryer) do() error {
+	for i := 0; i <= m.maxRetry; i++ {
+		m.process()
+		if !m.shouldRetry() {
+			return m.err
+		}
+
+		waitTime := m.getWaitTimeExp(i, 100)
+		time.Sleep(time.Millisecond * time.Duration(waitTime))
+	}
+
+	return ErrAPICallRejected
+}
+
+func (m *retryer) process() {
+	m.processFunc(m)
+}
+
+func (m *retryer) shouldRetry() bool {
+	if m.err != nil {
+		return false
+	}
+	if m.statusCode != http.StatusServiceUnavailable {
+		return false
+	}
+
+	r, err := parseNormalResponse(m.body)
+	if err != nil {
+		m.err = errors.Wrapf(err, "failed to parse normal response")
+		return false
+	}
+
+	return strings.HasPrefix(r.Message, "Please retry this request")
```

Comment:
Thanks for patching 🙏 

It seems that you are trying to determine if the request was rejected by the content of the response message, but [the response has an "isRejected" field in addition to the message](https://help.pixe.la/en/blog/release-request-rejecting).

If this is too difficult for you to use, please let me know!

Reply:
@a-know Thanks for letting me know.

I think it is better to use `isRejected` field rather than `message` fieald to determine if a request is rejected.
I will consider using `isRejected` field to determine if a request has been denied.
 
Thanks for the comment!

Reply:
@a-know I implemented it. #6

Thank you so much!